### ISSUE TITLE
is this still working? if it is any guides etc?

### ISSUE THREAD

**Issue by gugakatsi** [LukeZGD/LOVE-WrapLua#2]

**Comment by MrcSnm** [comment]
Yes, I probably was the last to use, I've made much work for the OneLua implementation for vita

**Comment by gugakatsi** [comment]
@MrcSnm thanks for reply , will test it out soon

**Comment by JoseOcasio** [comment]
It works, I recently did a snake game for the PSP to test this out. I had to modify a few things in the source of the wrapper since I was having multiple errors.
